src/Validator/Constraints/CartNotEmpty.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\ShopApiPlugin\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class CartNotEmpty extends Constraint
+{
+    /** @var string */
+    public $emptyCartMessage = 'sylius.shop_api.checkout.cart.empty';
```

Comment:
Can we be a little bit more coherent?
```suggestion
    public $message = 'sylius.shop_api.checkout.cart.empty';
```